### PR TITLE
Using federated auth for ACS tests

### PR DESCRIPTION
**Pull request by rajuanitha88**
Changes:
In order to move away from Secret based authentication, we want to adopt FIC.
For the same azure-sdk team has added the support and we need to use service connection for our live test pipelines to access our test resources.

### CODE REVIEW

**Review by LuChen-Microsoft — APPROVED** [review]

**Review by mjafferi-msft — APPROVED** [review]

**Review by maximrytych-ms — APPROVED** [review]

**Review by mjafferi-msft — APPROVED** [review]

**Review by mjafferi-msft — APPROVED** [review]

**Review by mikehang-msft — APPROVED** [review]

**Review by mikehang-msft — APPROVED** [review]

**Review by minwoolee-msft — APPROVED** [review]

File: sdk/communication/Azure.Communication.CallAutomation/tests.yml [hunk 1]
```diff
@@ -5,19 +5,18 @@ extends:
...
         SubscriptionConfigurations:
-          - $(sub-config-azure-cloud-test-resources)
           - $(sub-config-communication-services-cloud-test-resources-common)
           - $(sub-config-communication-services-cloud-test-resources-net)
-      Int:
-        SubscriptionConfigurations:
-          - $(sub-config-communication-int-test-resources-common)
-          - $(sub-config-communication-int-test-resources-net)
     Clouds: Public
     TestResourceDirectories:
```
Comment (weshaggard) [inline sdk/communication/Azure.Communication.CallAutomation/tests.yml:18]: If this is just the root now you can remove this entire setting as that is the default.

Comment (weshaggard) [reply sdk/communication/Azure.Communication.CallAutomation/tests.yml:18]: Also, how were these test deployments working before this change?

**Review by weshaggard — APPROVED** [review]

File: sdk/communication/Azure.Communication.CallAutomation/tests.yml (hunk 1 above)
Comment (rajuanitha88) [reply sdk/communication/Azure.Communication.CallAutomation/tests.yml:18]: I dont think it was for some of the other ACS SDKs. We fixed this as part of this work.

**Pull request merged**